GDAS-validation/run_job.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# run_job.sh
+# using rocotoboot, point to a global-workflow experiment
+# and run a specified job for a specified cycle
+
+usage() {
+  set +x
+  echo
+  echo "Usage: $0 -c /path/to/config.sh -t taskname"
+  echo
+  echo "  -c  path to config shell script to source"
+  echo "  -t  name of workflow task to run"
+  echo
+  exit 1
+}
+
+
+while getopts ":c:t:h" opt; do
+  case $opt in
+    c)
+      ConfigPath=$OPTARG
+      ;;
+    t)
+      TaskName=$OPTARG
+      ;;
+  esac
+done
+
+if [ $# -ne 4 ]; then
+  echo "Incorrect number of arguments"
+  usage
+fi
+
+# first, source the config script
+# this should be a shell script with the following variables defined
+#
+# EXPDIR=/path/to/EXPDIR/ do not include PSLOT
+# PSLOT="gdas_eval_satwind"
+# PDY=20210801
+# cyc=00
+#
+#
+source $ConfigPath
+
+ROCOTOEXP="-w ${EXPDIR}/${PSLOT}/${PSLOT}.xml -d ${EXPDIR}/${PSLOT}/${PSLOT}.db"
+# does the rocoto db file exist?
+if [ ! -f ${EXPDIR}/${PSLOT}/${PSLOT}.db ]; then
+  # run rocotorun just to generate it
+  rocotorun $ROCOTOEXP
+fi
+# let's check the status of the job you are attempting to run
+# just for your own information
+echo "==============================================================================="
+echo "==============================================================================="
+rocotorun $ROCOTOEXP
+rocotocheck $ROCOTOEXP -c ${PDY}${cyc}00 -t $TaskName
```

Comment:
Ugh, removing rocotorun results in _dependencies not satisfied_ messages.  Even though the previous job, say gdasanal, completed, the db file doesn't know this.   It thinks gdasanal is still _SUBMITTING_.  As a result, rocotocheck gdasanaldiag generates a `Dependencies are not satisfied` message.  This doesn't matter because the next thing `run_job.sh` does is rewind and reboot gdasanaldiag.   Given this, I suggest we remove this rocotocheck from the `run_job.sh`

Reply:
makes sense, I guess this is the problem with using rocoto in a way that it was not intended (but I think this is still preferred compared to making our own slurm batch submission tool)

Reply:
No changes required here.  Just a discussion with some points to capture in user readme.